### PR TITLE
Dl workshop event page

### PR DESCRIPTION
**Pull request by MaximilianFranz**
I Know the event image has wrong ratio, but i couldn't be fucked to make a new one :D

### CODE REVIEW

File: src/deeplearning.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    <link rel="icon" type="image/png" sizes="192x192"  href="img/favicon/android-icon-192x192.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="img/favicon/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="img/favicon/favicon-16x16.png">
+    <link rel="manifest" href="manifest.json">
+    <meta name="msapplication-TileColor" content="#A63324">
+    <meta name="msapplication-TileImage" content="images/favicon/ms-icon-144x144.png">
+    <meta name="theme-color" content="#A63324">
+
+    <title>Java 9, ein Über- und Ausblick - Hack & Söhne</title>
```
Comment (LeanderK) [inline src/deeplearning.html]: wrong title

File: src/deeplearning.html [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+      gtag('config', 'UA-90422115-1');
+    </script>
+
+    <!-- Heatmap.it Analytics -->
+    <script>
+        (function(h,e,a,t,m,p) {
+            m=e.createElement(a);m.async=!0;m.src=t;
+            p=e.getElementsByTagName(a)[0];p.parentNode.insertBefore(m,p);
+        })(window,document,'script','https://u.heatmap.it/log.js');
+    </script>
```
Comment (LeanderK) [inline src/deeplearning.html]: heatmap and analytics still present (you probably copied from master and not feater_dsgvo)

Comment (MaximilianFranz) [reply src/deeplearning.html]: thats right. Was surprised by that but didn't fix it

**Comment by MaximilianFranz** [comment]
Should be ready now

**Comment by LeanderK** [comment]
no github-notification 😕

**Comment by LeanderK** [comment]
@MaximilianFranz thanks for the changes! 👍

**Pull request merged**